easybuild/easyconfigs/t/termgraph/termgraph-0.3.1-python3.eb

```diff
@@ -0,0 +1,28 @@
+# jg (CSCS)
+easyblock = 'PythonPackage'
+
+name = 'termgraph'
+version = '0.3.1'
+_pymajver = '3'
+_pyshortver = '3.7'
+versionsuffix = '-python%s' % _pymajver
```

Comment:
```suggestion
versionsuffix = '-python%(pymajver)s'
```

Reply:
Can you poit me to your build dir please ? I get:
`== building and installing termgraph/0.3.1-python%(pymajver)s...`